wiki/Guides/Tablet_Purchase/en.md

```diff
@@ -46,24 +46,39 @@ At the time of writing there are three versions of the Bamboo, all confusingly u
 
 ![](img/gen1.jpg "First generation Bamboo (Pen and Touch, Touch, and Fun - Bamboo Fun shown)")
 
-![](https://puu.sh/dlQH "Second generation Bamboo (Pen and Touch, Pen, Touch, Fun, and Craft - Bamboo Pen and Touch shown)")
+![](img/gen2.jpg "Second generation Bamboo (Pen and Touch, Pen, Touch, Fun, and Craft - Bamboo Pen and Touch shown)")
```

Comment:
I'm pretty sure these images are missing in the PR?

Reply:
it looks like they were there all the time (since 4 years ago, yeah)

Reply:
Appear just fine on mine and https://github.com/ppy/osu-wiki/tree/master/wiki/Guides/Tablet_Purchase/img
![IMG_20210913_161654](https://user-images.githubusercontent.com/76740201/133058347-0a2b3186-4379-478a-bbb8-ccf661bd8605.jpg)

Reply:
Ah, right. Thought they were new ones.